js/src/main/scala/org/portablescala/reflect/Reflect.scala

```diff
@@ -1,9 +1,13 @@
 package org.portablescala.reflect
 
+import com.github.ghik.silencer.silent
+
 import scala.scalajs.reflect.{Reflect => SJSReflect}
 
 object Reflect {
-  /** Reflectively looks up a loadable module class.
+
+  /**
+   * Reflectively looks up a loadable module class.
```

Comment:
This is quite obviously wrong.

Reply:
Not obvious to me, since I have almost zero knowledge about Scala.js (this is the JS part of Reflect, right?) :grin: 

Or did you mean the comment formatting?

Reply:
yes, I meant the formatting :) (see too terse).

the lower comments are indented by more. further, I think this repo follows Scala.js style:
https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md

There have been quite some attempts to auto-format it, often the formatters were too buggy. Last attempt:
https://github.com/scala-js/scala-js/pull/4260

Reply:
Now I see what's going on.
The Scala.js guideline says it should be
```scala
/** Reflectively ...
```
But Scalafmt keeps changing it to
```scala
/**
 * Reflectively 
```
even though we have `preset=Scala.js`. That's a bit annoying, I have to say :confused: But IMHO still much better than no automatic Scalafmt formatting at all :+1:

Reply:
The "Scala.js" preset was defined 5 years ago when scalafmt was in its infancy. It is completely obsolete and several more options are required to get close to the Scala.js style. You can have a look at the most recent PRs trying to enable scalafmt in the core Scala.js codebase for inspiration.

Reply:
I see! Reformatted :heavy_check_mark: